src/test/test_utils.h

```diff
@@ -0,0 +1,60 @@
+#ifndef TEST_TEST_UTILS_H_
+#define TEST_TEST_UTILS_H_
+
+#include "database/cost_matrix_database.h"
+#include "database/online_database.h"
+#include "localization_protos.pb.h"
+
+#include "gtest/gtest.h"
+
+#include <filesystem>
+#include <fstream>
+#include <string>
+#include <vector>
+
+namespace test {
```

Comment:
nit: if I'm being very picky, I would also have it all wrapped in another namespace for the whole project to make sure that you're not clashing with any other namespace from your third-party includes.

Reply:
done.